client/components/MessageTypes/error.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+	<span class="content">
+		<ParsedMessage :network="network" :message="message" :text="errorMessage" />
+	</span>
+</template>
+
+<script>
+import ParsedMessage from "../ParsedMessage.vue";
+
+export default {
+	name: "MessageTypeError",
+	components: {
+		ParsedMessage,
+	},
+	props: {
+		network: Object,
+		message: Object,
+	},
+	computed: {
+		errorMessage() {
+			const reason = this.message.reason;
+			const errorType = this.message.error;
+			const nick = this.message.nick;
+			const channel = this.message.channel;
```

Comment:
Can you remove these consts and just reference them directly, as each error message uses different properties.